tests/e2e/operator/operator_test.go

```diff
@@ -358,8 +331,17 @@ func log(a ...any) {
 	GinkgoWriter.Println(a...)
 }
 
+func getCRDsName() []string {
```

Comment:
`crdList` should be a parameter instead of using the global variable. From the outside, it's not really clear what this function is doing (where it's getting the CRDs).